composer.json

```
@@ -14,6 +14,7 @@
 
         "amazon",
         "asana",
+        "apple",
```

This is an alphabetically ordered list:
```suggestion
        "apple",
        "asana",
```

---

Thanks. Its fixed now